src/tests/Deatails.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Details from './Details';
+
+const mockCountry = {
+  country: 'Test Country',
+  population: '1000000',
+  continent: 'Test Continent',
+  recovered: '200000',
+  deaths: '5000',
+  tests: '10000',
+  active: '100000',
+  critical: '1000',
+};
+
+it('should render the country details', () => {
+  render(<Details country={mockCountry} />);
+
+  const countryTitle = screen.getByRole('heading', { Name: 'Test Country' });
+  expect(countryTitle).toBeInTheDocument();
+});
```

Comment:
You have done well to add some tests. As mentioned in the [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/be3b13a85ca425c0bf0eeb1f130b6a52e36deb4f/capstone/react_capstone.md?plain=1#L76), kindly include some unit tests for some pure functions in Redux code.